doc/api/errors.md

```diff
@@ -364,13 +364,8 @@ detailed [here](#errors_system_errors).
 
 ## Class: AssertionError
 
-A subclass of `Error` that indicates the failure of an assertion. Such errors
-commonly indicate inequality of actual and expected value.
-
-```js
-assert.strictEqual(1, 2);
-// AssertionError [ERR_ASSERTION]: 1 === 2
-```
+A subclass of `Error` that indicates the failure of an assertion. For further
```

Comment:
Microscopic nit that can totally be ignored if you want: The word `further` can be omitted. For that matter, so can all of `For further details,`. It can just be ``` See [`Class: AssertionError`][]. ```